README.md

```diff
@@ -1,53 +1,68 @@
-## comskip
+## Comskip
 
-commercial detector
+Commercial detector
 http://www.kaashoek.com/comskip/
 
-### requirements
+### Requirements
 
-- recent build of ffmpeg with headers and shared libraries (2.4+)
+- FFmpeg with headers and shared libraries (2.4+)
 - libargtable2 for option parsing
-- optional: sdl for gui on linux/mac
+- Optional: SDL for building the Comskip GUI
 
-### building
+### Building
 
-#### windows
+#### Building from GitHub
+
+Building directly from GitHub also requires GNU autotools (autoconf, automake, and libtool).
 
 ```
 $ git clone git://github.com/erikkaashoek/Comskip
 $ cd Comskip
-$ mingw32-make CC=mingw32-gcc INCLUDES="-I./ffmpeg/include" LIBS="-L./ffmpeg/lib"
-
-$ comskip.exe
+$ ./autogen.sh
+$ ./configure
+$ make
 ```
 
-#### macosx
+#### Building from a tarball
```

Comment:
Packagers typically do, though, so anyone working with a source .deb or .rpm will need the different instructions. I don't see any harm in leaving that bit in the readme.